File: examples/kotlin/src/test/kotlin/InstallDriversTest.kt
import io.github.bonigarcia.wdm.WebDriverManager
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import org.openqa.selenium.WebDriver
import org.openqa.selenium.chrome.ChromeDriver
import org.openqa.selenium.edge.EdgeDriver
import org.openqa.selenium.firefox.FirefoxDriver
import org.openqa.selenium.ie.InternetExplorerDriver

class InstallDriversTest {
    @Test
    fun chromeSession() {
        WebDriverManager.chromedriver().setup()
        val driver: WebDriver = ChromeDriver()
        driver.quit()
    }

    @Test
    fun edgeSession() {
        WebDriverManager.edgedriver().setup()
        val driver: WebDriver = EdgeDriver()
        driver.quit()
    }

    @Test
    fun firefoxSession() {
        WebDriverManager.firefoxdriver().setup()
        val driver: WebDriver = FirefoxDriver()
        driver.quit()
    }

    @Disabled("Only runs on Windows")
    @Test
    fun ieSession() {
        WebDriverManager.iedriver().setup()
        val driver: WebDriver = InternetExplorerDriver()
        driver.quit()
    }
}

File: examples/dotnet/SeleniumDocs/GettingStarted/InstallDriversTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Edge;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.IE;
using SeleniumDocs.TestSupport;
using WebDriverManager;
using WebDriverManager.DriverConfigs.Impl;

namespace SeleniumDocs.GettingStarted
{
    [TestClassCustom]
    public class InstallDriversTest
    {
        [TestMethod]
        public void ChromeSession()
        {
            new DriverManager().SetUpDriver(new ChromeConfig());

            var driver = new ChromeDriver();

            driver.Quit();
        }

        [TestMethod]
        public void EdgeSession()
        {
            new DriverManager().SetUpDriver(new EdgeConfig());

            var driver = new EdgeDriver();

            driver.Quit();
        }

        [TestMethod]
        public void FirefoxSession()
        {
            new DriverManager().SetUpDriver(new FirefoxConfig());

            var driver = new FirefoxDriver();

            driver.Quit();
        }

        [EnabledOnOs("WINDOWS")]
        [TestMethod]
        public void InternetExplorerSession()
        {
            new DriverManager().SetUpDriver(new InternetExplorerConfig());

            var driver = new InternetExplorerDriver();

            driver.Quit();
        }
    }
}